### PR TITLE
feat: uniform my-items page and widgets popup

### PR DESCRIPTION
**Pull request by ArtemLapkin**

### CODE REVIEW

File: admin/js/menu-page.js [hunk 1]
```diff
@@ -1,159 +1,25 @@
...
-            $(self.cache.$modalShortcodeTextarea).val('')
-          })
-        }
-      })
-      self.cache.$closeModalButton.on('click', function (e) {
-        self.cache.$modal.fadeOut(function () {
-          $(self.cache.$modalShortcodeTextarea).val('')
-        })
-      })
+  if( $('[data-opinionstage-content-popup-template]').length > 0 ) {
```
Comment (kucaahbe) [inline admin/js/menu-page.js:2]: let's fix indentiation here

File: admin/js/menu-page.js [hunk 2]
```diff
@@ -1,159 +1,25 @@
...
-        $("[" + t + "]")[0].select()
-        document.execCommand("copy")
+    $('#opinionstage-dialog-close').on('click', function (e) {
+      e.preventDefault()
+      $modal.fadeOut(function () {
+          $(this).find('#opinionstage-widget-shortcode').val('')
       })
-    },
+    })
+  OpinionStage.contentPopup.renderMyItemsPageView(showModal)
```
Comment (kucaahbe) [inline admin/js/menu-page.js:16]: on my items page, it's not a content popup anymore, further usage of this naming will lead to confusion, suggest we use another naming and implement my items page and content popup as separate entry points: `OpinionStage.widgetList.show(...)` for my items page and `OpinionStage.contentPopup.open()`

File: admin/content-popup-template.html.php [hunk 3]
```diff
@@ -23,21 +24,30 @@
...
-			<div class='header__container'>
+			<div class='header__container <?php echo $is_my_items_admin_page ? 'mw-1000' : ''; ?>'>
 				<div class='header__logo'>
 					<a href='<?php echo esc_url( OPINIONSTAGE_SERVER_BASE ); ?>' target='_blank'>
 						<img src='<?php echo esc_url( plugins_url( 'admin/images/os-logo-header.png', plugin_dir_path( __FILE__ ) ) ); ?>'>
 					</a>
 				</div>
 				<div class='header__action'>
-					<div class='btn-close' @click="closePopup">&#x2715;</div>
+					<?php if ( $is_my_items_admin_page ) { ?>
```
Comment (kucaahbe) [inline admin/content-popup-template.html.php:34]: conditionals in shared implementation is a strong sign of incorrect architectural direction, let's discuss this in details

**Comment by ArtemLapkin** [comment]
Hey @kucaahbe, I updated PR, take a look please if all is ok - I will squash all commits into one. Thx!

**Review by kucaahbe — APPROVED** [review]

**Pull request merged**